packages/react-server/core/components/RootElement.js

```diff
@@ -23,7 +23,17 @@ class RootElement extends React.Component {
 
 				// Okay, now we've complained about it
 				// sufficiently, let's go ahead and update.
-				this.props.childProps = childProps;
+
+				// Because this.props.childProps is writable/configurable:false
+				// we update the entirety this.props through a clone
+				var propstemp = {};
+				for (var k in this.props) {
```

Comment:
Actually, this could all just be a one-liner with `_.assign`:

```javascript
this.props = _.assign({}, this.props, {childProps});
```

We're already using the `assign` method from lodash elsewhere, so this won't increase our bundle size.

Our pattern for including lodash methods is:

```javascript
const _ = {
        assign: require('lodash/assign'),
};
```